packages/theme/package.json

```diff
@@ -30,6 +30,9 @@
     "vitest": "^0.17.1"
   },
   "oclif": {
-    "commands": "dist/cli/commands"
+    "commands": "dist/cli/commands",
+    "hooks": {
+      "prerun": "dist/hooks/prerun"
```

Comment:
That won't work with the logging to file as designed currently. `cli-kit` is agnostic of where it's being run, so it can't specify a logfile.

If we get rid of multiple logfiles (which is under discussion), it'll be possible to push this into `cli-kit` as suggested.